nss-day-cohort-41/tabloid-fullstack-first-amendment-dancers-andy
# AK Category Tickets

# Description
Full CRUD for Categories in API and Front-end
Can see all Categories (except special "other" category) in Category management >>List
Can edit categories or delete categories with option to complete or go back

Fixes # (9, 10, 11, 12)
## Type of change
Please delete options that are not relevant.


# Testing Instructions

``Git fetch origin ak-category``
``Git checkout origin ak-category``
After Login, select "Category Management"
Should display all available categories with Link to ADD a new Category
When Adding a new category must enter a Name then select Add Category button
will redirect back to List with new category
if you select edit or delete button it will redirect page to edit or delete confirming you wish to complete action
or select Go Back to redirect back to list


# Checklist:
